README.md

```diff
@@ -1,77 +1,83 @@
 
-## Supported Verticals
+> **🚀 Industry Benchmark**: Spanning 20 essential verticals <em>(soon)</em>, our taxonomy encompasses categories, attributes, and values, all thoughtfully integrated within Shopify and numerous marketplaces.
+
+<p align="right"><em>See more on <a href="https://help.shopify.com/manual/products/details/product-category">help.shopify.com</a></em></p>
+
+<!-- omit in toc -->
+## 🗂️ Table of Contents
+
+- [🕹️ Interactive explorer](#️-interactive-explorer)
+- [📚 Taxonomy details](#-taxonomy-details)
+- [🗄️ How we organize things](#️-how-we-organize-things)
+  - [📦 Integrating with the taxonomy](#-integrating-with-the-taxonomy)
+  - [🧑🏼‍🏫 Changing the taxonomy](#-changing-the-taxonomy)
+  - [👩🏼‍💻 Developing the system](#-developing-the-system)
+- [📅 Releases](#-releases)
+- [📜 License](#-license)
+
+## 🕹️ Interactive explorer
+
+Ready to dive in? [Explore our taxonomy interactively](https://shopify.github.io/product-taxonomy/?categoryId=sg-4-17-2-17) now to visualize and discover the potential of our taxonomy.
+
+## 📚 Taxonomy details
 
 Verticals will be regularly released to this repository, with all 20 published by the end of February, 2024. The following table shows the current status of each vertical.
 
 | Vertical | Status |
 |----------|----------|
-| Apparel & Accessories | ✅ |
-| Food, Beverages, & Tobacco | ✅ |
-| Home & Garden | ✅ |
-| Sporting Goods | ✅ |
-| Furniture | Coming soon [next] |
-| Health & Beauty | Coming soon [next] |
-| Animals & Pet supplies | Coming soon |
-| Electronics | Coming soon |
-| Media | Coming soon |
-| Arts & Entertainment | Coming soon |
-| Vehicles & parts | Coming soon |
-| Toys & games | Coming soon |
-| Luggage & bags | Coming soon |
-| Software | Coming soon |
-| Cameras & optics | Coming soon |
-| Hardware | Coming soon |
-| Baby & toddler | Coming soon |
-| Business & industrial | Coming soon |
-| Office supplies | Coming soon |
-| Services | Coming soon |
-
-## Organization
-
-This github repository includes both the source code for defining the taxonomy, as well as distribution files for consuming the taxonomy. We welcome public input to evolve and adjust this taxonomy. Requested changes will be reviewed and considered for subsequent version updates.
+| Apparel & Accessories | ✅ done |
+| Food, Beverages, & Tobacco | ✅ done |
+| Home & Garden | ✅ done |
+| Sporting Goods | ✅ done |
+| Furniture | 🚀 next |
+| Health & Beauty | 🚀 next |
+| Animals & Pet supplies | 💁🏼‍♀️ soon |
+| Electronics | 💁🏼‍♀️ soon |
+| Media | 💁🏼‍♀️ soon |
+| Arts & Entertainment | 💁🏼‍♀️ soon |
+| Vehicles & parts | 💁🏼‍♀️ soon |
+| Toys & games | 💁🏼‍♀️ soon |
+| Luggage & bags | 💁🏼‍♀️ soon |
+| Software | 💁🏼‍♀️ soon |
+| Cameras & optics | 💁🏼‍♀️ soon |
+| Hardware | 💁🏼‍♀️ soon |
+| Baby & toddler | 💁🏼‍♀️ soon |
+| Business & industrial | 💁🏼‍♀️ soon |
+| Office supplies | 💁🏼‍♀️ soon |
+| Services | 💁🏼‍♀️ soon |
 
-### Distribution files
+## 🗄️ How we organize things
```

Comment:
Something about the sub-headers is not resonating. MaybeL

## 🗄️ How we organize things --> How to use this GitHb repo

### 📦 Integrations - How to integration with the taxonomy

### 📦 Making Changes - How to submit changes to  the taxonomy